### PR TITLE
UC-208 Removed all mentions of the Special:FacebookConnect page

### PR DESCRIPTION
**Pull request by garthwebb**
Also removed unused messages that were part of this page.

https://wikia-inc.atlassian.net/browse/UC-208

### CODE REVIEW

**Comment by jsutterfield** [comment]
LGTM.

File: extensions/wikia/FacebookClient/SpecialFacebookConnectController.class.php [hunk 1]
```diff
@@ -1,280 +0,0 @@
...
-		$userNameCookie = $wg->CookiePrefix . 'UserName';
-		$userName = isset( $_COOKIE[$userNameCookie] ) ? trim( $_COOKIE[$userNameCookie] ) : '';
-
-		$errorMessage = $wg->Request->getVal( 'errorMessage' );
-
-		if ( $errorMessage ) {
-			$this->errorMessage = wfMessage( $errorMessage )->plain();
-		}
-		$this->formAction =  SpecialPage::getSafeTitleFor( 'FacebookConnect' )->getLocalURL();
-		$this->legendText = wfMessage( 'fbconnect-connect-login-legend' )->text();
```
Comment (armonr) [inline extensions/wikia/FacebookClient/SpecialFacebookConnectController.class.php]: Please also remove `fbconnect-connect-login-legend` in i18n file.

Comment (armonr) [inline extensions/wikia/FacebookClient/SpecialFacebookConnectController.class.php]: And `fbconnect-connect-username-label` ... and `fbconnect-connect-password-label`

File: extensions/wikia/FacebookClient/FacebookClient.setup.php [hunk 2]
```diff
@@ -34,7 +34,6 @@
 $wgAutoloadClasses['FacebookMapModel'] =  $dir . 'FacebookMapModel.class.php';
 $wgAutoloadClasses['FacebookClientFactory'] =  $dir . 'FacebookClientFactory.php';
 $wgAutoloadClasses['FacebookClientController'] =  $dir . 'FacebookClientController.class.php';
-$wgAutoloadClasses['SpecialFacebookConnectController'] =  $dir . 'SpecialFacebookConnectController.class.php';
```
Comment (armonr) [inline extensions/wikia/FacebookClient/FacebookClient.setup.php]: Line 66 of this file should also be removed https://github.com/Wikia/app/pull/6026/files#diff-ac942f59224da105aff611a08a8dfe5dR66

Comment (armonr) [inline extensions/wikia/FacebookClient/FacebookClient.setup.php]: And `fbconnect-preferences-connected-error` is not being used anymore, which is my bad for not removing it :/

**Comment by armonr** [comment]
LGTM

**Comment by garthwebb** [comment]
Fixed merge conflicts

**Pull request merged**